DidenkoSerhiy/HW4/homework_4.py

```diff
@@ -0,0 +1,8 @@
+c = float(input("Input the temperature in Celcius: "))
```

Comment:
good job but 
use more meaningful names for variables like this
celsius = float(input("Input the temperature in Celsius: "))
fahrenheit = celsius * 9/5 + 32

if celsius > -273.15:
    print(f"{celsius}\u00b0C is equivalent to {fahrenheit}\u00b0F")
elif celsius <= -273.15:
    print("Error: Temperature below absolute zero -273.15\u00b0C")

Reply:
Noted, thanks